setup.py

```diff
@@ -21,7 +21,7 @@
 
 setup(
     name='strato_query',
-    version='3.10.0',
```

Comment:
Could we just make this a patch 3.10.1 please?

Reply:
Yes, updated.